.changeset/rich-foxes-deliver.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/cloudflare': patch
+---
+
+Improves the developer experience, when using Cloudflare runtime with `astro dev`
```

Comment:
t ```suggestion
Adds a new property `persistTo`, which allows .... when using Cloudflare runtime with `astro dev`
```
You didn't think I was going to let you get away with that, did you?  :smile:

Also, isn't there a breaking change in here because a value's format has changed from a string to an object?

Reply:
I updated the Changeset, but would like for you to check it from the language aspect again. :)